api/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/hcpclusters/preview/2024-06-10-preview/openapi.json

```diff
@@ -1462,6 +1462,16 @@
             "create"
           ]
         },
+        "diskEncryptionSetId": {
+          "type": "string",
+          "format": "uri",
+          "description": "The ID of the DiskEncryptionSet resource to use to encrypt the OS disks for the VMs.\nCan be used with either platform (Azure) managed, or customer managed encryption keys.\nThis needs to exist in the same subscription id listed in the Hosted Cluster, HostedCluster.Spec.Platform.Azure.SubscriptionID.\nDiskEncryptionSetID should also exist in a resource group under the same subscription id and the same location\nlisted in the Hosted Cluster, HostedCluster.Spec.Platform.Azure.Location.\n\nDetails on how to create a Disk Encryption Set can be found here:\nhttps://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-customer-managed-keys-portal#set-up-your-disk-encryption-set",
+          "maxLength": 285,
+          "x-ms-mutability": [
```

Comment:
Pending to define this

Reply:
Waiting on Hypershift to tell me if we can PATCH this. Thread https://redhat-external.slack.com/archives/C075PHEFZKQ/p1751380214598679